content/week-two/index.md

```diff
@@ -0,0 +1,15 @@
+---
+title: Week Two

```

Comment:
Week two isn't a great title. One of Tilde's superpowers is self paced learning. 

What should the learner achieve by going through this content? Maybe a better title is "Challenge: Improve working professionals' experiences of working remotely"